playbooks/robusta_playbooks/job_restart_on_oomkilled.py

```diff
@@ -50,49 +52,57 @@ def job_restart_on_oomkilled(event: JobEvent, params: IncreaseResources):
             index = ind
             status_flag = True
             break
+    # Extracting request['memory'] from the containers
+    max_res = PodContainer.get_requests(
+        job_event.spec.template.spec.containers[index]
+    ).memory
 
-    # Extracting request['memory'] from the containers and comparing with max_resource
-    max_res, mem = split_num_and_str(
-        job_event.spec.template.spec.containers[index].resources.requests["memory"]
-    )
-    if float(max_res) < params.max_resource:
-
-        if status_flag:
-            job_spec = restart_job(job_event, params.increase_to)
+    if status_flag:
+        if float(max_res) < params.max_resource:
+            job_spec = restart_job(job_event, params.increase_by, params.max_resource)
 
             job_temp = job_spec.spec.template.spec.containers[index].resources.requests[
                 "memory"
             ]
             finding.add_enrichment(
                 [
                     MarkdownBlock(
-                        f"*Job Restarted With Memory Increament*\n```\n{job_temp}\n```"
+                        f"*Job Restarted With Memory Increment*\n```\n{job_temp}\n```"
```

Comment:
can we say which container OOM crashed and specify the increased? e.g "Job named X crashed because of OOM, increasing container <CONTAINER NAME> memory from X<UNIT> to Y<UNIT>"